src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -239,13 +239,14 @@ const PagedTable: FC<Props> = ({result, properties}) => {
     }
 
     let timeout
+    let animationFrameID
     const resizer = new ResizeObserver(entries => {
       if (timeout) {
         clearTimeout(timeout)
       }
 
       timeout = setTimeout(() => {
-        requestAnimationFrame(() => {
+        animationFrameID = requestAnimationFrame(() => {
           setHeight(entries[0].contentRect.height)
         })
       }, 200)
```

Comment:
do you know why this RAF is in a `setTimeout`?

Reply:
There is a general pattern when using ResizeObserver --> to then trigger a callback (e.g. set height of inner component) which then may recursively trigger ResizeObserver again. This is a known problem, which is why browsers do not surface these errors in the console...but it does still gets report in `window.onerror()` --> which we pipe to honeybadger.

Here is an issue ticket from WICG, which includes this^^ conclusion, as well as many code samples where a community of devs have listed all the code patterns which they use to decrease the number of recursive triggers (although is impossible to eliminate). One of the patterns is to have the callback action be encapsulated in a `setTimeout()` (`https://github.com/WICG/resize-observer/issues/38#issuecomment-422085926`), which queues it up at the back of the javascript runtime macrotask queue.

Reply:
>One of the patterns is to have the callback action be encapsulated in a `setTimeout()` which queues it up at the back of the javascript runtime macrotask queue.

Thanks for the explanation. Is the 200 millisecond delay significant? Could this could this be achieved with a delay of 1 or 0 milliseconds? The example doesn't set the delay value explicitly, which defaults it to a value of 0

Reply:
The timeout was put in previously. I can only guess the 200ms is arbitrary. The 0 is sufficient for insertion at the back of the macrotask queue.